packages/experimental/Button/src/Button.types.ts

```diff
@@ -76,6 +76,7 @@ export interface ButtonTokens extends ButtonCoreTokens {
 export interface ButtonCorePropsWithInnerRef extends Omit<IWithPressableOptions<ViewProps>, 'onPress'> {
   /*
    * Text to show on the Button.
+   * Only used in FAB
```

Comment:
Should be removed soon, so just going to leave this comment instead making a whole type for it.